lib/modules/manager/asdf/readme.md

```diff
@@ -3,8 +3,45 @@ Keeps the [asdf](https://asdf-vm.com/manage/configuration.html#tool-versions)
 
 Because `asdf` supports the version management of many different tools, specific tool support needs to be added one by one.
 
-Only the following tools are currently supported
+The following tools are currently supported:
```

Comment:
You'll have to fix the merge conflicts in this file. 😉

Reply:
merged :+1: